docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -717,152 +740,123 @@ Date: Thu, 23 Aug 2018 11:57:17 GMT
 
 [{"id":1,"name":"Secretlessly Fluffy"}]
 </pre>
-We expect the request to respond with a JSON array containing all the pets.
 
-There we have it. This application is communicating with a target service without managing any secrets.
+We should get a 200 response with a JSON array of the pets.
 
-## Rotate Target Service Credentials
+That's it! 
 
-In this section, you get to see how an application using the Secretless Broker deals with credential rotation.
-These are the steps you wil take to rotate the credentials of the dabatase:
-+ rotate application-user DB-credentials in the database
-+ update the application-user DB-credentials in the vault
-+ prune existing connections established using old credentials
+<div class="the-big-finish">
+  The application is connecting to a password-protected Postgres database
+  <b>without any knowledge of the credentials</b>.
 
-### Poll Application API [separate terminal]
-Before rotating, **you will run the commands in this section in a new and separate terminal** to poll the retrieve pets endpoint (`GET /pets`). This will allow you to see the request-response cycle of the application. If something goes wrong, like a database connection failure you will see it as a > 400 HTTP status code.
+  <img src="/img/its_magic.jpg" alt="It's Magic"/>
+</div>
 
```

Comment:
Maybe we should have a Call to Action here? We can do one of those big blue buttons and direct them to our docs on deploying.

> Interested in learning more? Check out our [documentation](https://docs.secretless.io/Latest/en/Content/docs/get_started/configuration.htm) to get started deploying Secretless Broker for your application!

Reply:
or something like that. I like the idea of a big blue button like at the top somehow.

Reply:
done